src/utils.ts

```diff
@@ -57,3 +58,40 @@ export function callEachWith (callbacks: Function[], arg0?: any) {
     cb(arg0)
   }
 }
+
+const isBrowser = typeof navigator !== 'undefined'
```

Comment:
I think mostly (including nuxt) `typeof window` is used to check browser env and replaced with `false`. Right now there is no tree-shaking benefit but unless you have specific reason to use navigator, i think window would be better...